### ISSUE TITLE
RichTextEditorMentionListbox test intermittent with error No selection range exists

### ISSUE THREAD

**Issue by rajsite** [ni/nimble#2219]
# 🐛 Bug Report

The test `RichTextEditorMentionListbox Dynamically update mention popup items based on configuration changes should show mention popup for multiple mention configuration elements FAILED` is intermittent and fails with the error message:

```
InvalidStateError: Selection.collapseToEnd: No selection range exists thrown
```

Update: same error seen on test `Markdown serializer Serialize rich text editor content to its respective markdown Mention node` in [this build](https://github.com/ni/nimble/actions/runs/9897035682/job/27340606528?pr=2242). That test is now disabled as well.

## 💻 Repro or Code Sample

See build https://github.com/ni/nimble/actions/runs/9686770115/job/26729936486#step:17:411

## 💁 Possible Solution

Seems like it may be related to a similar sounding error from https://github.com/ni/nimble/pull/2200

**Comment by rajsite** [comment]
@vivinkrishna-ni could you create a corresponding AzDo issue to track this intermittent test failure? It seems like it may be similar to the type of failure covered in https://github.com/ni/nimble/pull/2200.

**Comment by vivinkrishna-ni** [comment]
Created an AzDo bug: [Bug 2787273](https://dev.azure.com/ni/DevCentral/_workitems/edit/2787273): Intermittent failures in Rich Text Editor for Mention support

Last time (in https://github.com/ni/nimble/pull/2200), it was evident that the editor was not focused at the time of committing the mention node and was failing in local with some conditions. However, this time, it is a rare occurrence where I tested like running the tests in a loop for about a thousand times, running the test that ran before in the failed pipeline, or keeping the breakpoint at the line where the error was thrown in Tiptap, which still not be able to reproducible.

**Comment by rajsite** [comment]
@vivinkrishna-ni an additional test failed with the same error message and was disabled in https://github.com/ni/nimble/pull/2262

**Comment by vivinkrishna-ni** [comment]
Closing the issue as the test failures are fixed as part of #2403. Please feel free to reopen the issue if the same tests are failing again.
